demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -687,58 +622,50 @@ <h5>Simulcast Encoding Parameters</h5>
           <button id="button-content-share" type="button" class="btn btn-success" title="Toggle content share">
             <%= require('../../node_modules/open-iconic/svg/camera-slr.svg') %>
           </button>
-          <div class="btn-group" role="group">
-            <button id="button-content-share-drop" type="button" class="btn btn-success dropdown-toggle dropdown-toggle-split"
-                    data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"
-                    title="Select content to share"></button>
-            <div id="dropdown-menu-content-share" class="dropdown-menu dropdown-menu-end"
-                 aria-labelledby="button-content-share-drop" x-placement="bottom-start"
-                 style="position: absolute; transform: translate3d(0px, 38px, 0px); top: 0px; left: 0px; will-change: transform;">
-              <a id="dropdown-item-content-share-configuration" class="dropdown-item" href="#">Configurations</a>
-              <a id="dropdown-item-content-share-screen-capture" class="dropdown-item" href="#">Screen Capture...</a>
-              <a id="dropdown-item-content-share-screen-test-video" class="dropdown-item" href="#">Test Video</a>
-              <a id="dropdown-item-content-share-test-mono-audio-speech" class="dropdown-item" href="#">Test Mono
-                Audio Speech</a>
-              <a id="dropdown-item-content-share-test-stereo-audio-speech" class="dropdown-item" href="#">Test Stereo
-                Audio Speech</a>
-              <a id="dropdown-item-content-share-test-stereo-audio-tone" class="dropdown-item" href="#">Test Stereo
-                Tone (L-500Hz R-1000Hz)</a>
-              <a id="dropdown-item-content-share-file-item" class="dropdown-item" href="#"><input
-                      id="content-share-item" type="file"></a>
-              <a id="dropdown-item-content-share-stop" style="display:none" class="dropdown-item" href="#">Stop
-                Sharing</a>
-            </div>
+          <button id="button-content-share-drop" type="button" class="btn btn-success dropdown-toggle dropdown-toggle-split"
+                  data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"
+                  title="Select content to share"></button>
+          <div id="dropdown-menu-content-share" class="dropdown-menu dropdown-menu-end"
+                aria-labelledby="button-content-share-drop" x-placement="bottom-start"
+                style="position: absolute; transform: translate3d(0px, 38px, 0px); top: 0px; left: 0px; will-change: transform;">
+            <a id="dropdown-item-content-share-configuration" data-bs-toggle="modal"
+            data-bs-target="#content-share-configuration-modal" class="dropdown-item" href="#">Configure Content Sharing</a>
+            <a id="dropdown-item-content-share-pause-resume" style="display:none" class="dropdown-item" href="#">Pause Content Share</a>
+            <h6 class="dropdown-header">Content Share Source</h6>
```

Comment:
This looks weird from the screenshot. If it is a header then it should appear more highlighted from the rest of the option.

Reply:
Yes totally agree I've been bothered by bootstrap in the past in how the drop down headers look. Let me try messing with the styling.

Reply:
I gave up, replaced it with a divider instead which i think is clear enough.

Reply:
That is fine but could you update the screenshot in the PR description :D

Reply:
Done!